### DISCUSSION TITLE
Die Verlorenen

### DISCUSSION THREAD

**Discussion by ImTander** [nzp-team/nzportable#686] [Map Releases]
Welcome to my first ever custom map! 

Inspired by some of the smaller more difficult maps in the series, 
I tried to make a map that felt like a classic zombies map with a small & loose story that you can piece together by looking at things around the map.

If you find any major bugs or glitches, waypoint issues, etc. Please let me know! again my first map so I would like to try to fix them.

Personally played and tested on PC/PS Vita.

Map Features : 

- Unlockable PaP + EE song
- 4 core perks + random perk machine
- Buyable ending

![Base Profile Screenshot 2024 03 19 - 17 54 07 05](https://github.com/nzp-team/nzportable/assets/161973907/b98147af-9ec9-4830-9d86-ce979a5de587)
![Base Profile Screenshot 2024 03 19 - 19 24 27 84](https://github.com/nzp-team/nzportable/assets/161973907/2a317a9c-9e31-4848-96ac-8046200476f0)
![dsc1](https://github.com/nzp-team/nzportable/assets/161973907/8d6c2f1b-8e00-43f6-8787-a5db6ffeeaaf)

Thank you to everyone in the Discord for your help!

# [DOWNLOAD](https://archive.org/download/die-verlorenen_202403/Die%20Verlorenen.zip)

**Comment by akumasjack** [comment]
This map is fun! It has the best first room out of all the maps I've played and I liked the tease of seeing the perks behind the barriers. I will note that in the first room you can see the zombies spawning through the wall near the shotgun wall buy, also the zombies run into the shelves in the second room and you can kind of camp near the thompson because of that

**Reply by ImTander** [reply]
thanks for trying it out! The small spaces are a little more narrow then they should be, making it easy to camp and that was an oversight on my part not being fully familiar with the scale of things. I'll def take the time on future maps to make sure things are properly spaced more so the zombies can navigate a bit better!

**Comment by Justsomerandom3dsuser** [comment]
map is fire although hard to play cause 3ds but ill get the hang of it lol

**Comment by Naievil** [comment]
Looks like a lot of fun!

**Comment by MuzakyGood** [comment]
Does the map work on PSP too?

**Comment by ImTander** [comment]
**( UPDATE : 1/23/24 )**

-Added perk jingles 
-Adjusted Double Tap perk machine model to Double Tap 1.0

**Comment by Jeyphr** [comment]
Be sure to check out my review ( when it's out ) : https://youtu.be/3qZQImlKnWI

**Reply by Jeyphr** [reply]
It's out now!

**Comment by M0THRAK** [comment]
Hey, I've went ahead and redid the waypoints for this map, if anyone is interested in using them. This doesn't fix all of the path finding issues in the map, however at the very least it fixes the waypointing errors.

Download them here: [Updated Die Verlorenen Waypoints](https://archive.org/download/updated-die-verlorenen-waypoints/Updated%20Die%20Verlorenen%20Waypoints.zip)

**Comment by ProfessorZIRE** [comment]
Is the random perk machine just a wunderfizz knockoff?

**Comment by ProfessorZIRE** [comment]
BRUHHH, archive.org ain't working for me, maybe cos im using a vpn cos im at school and its blocked

**Comment by nonMat** [comment]
Oh wow this map is amazing!!!
First things first the layout and the way it's decorated is amazing!!!
I played it blind and the teddys were really fun to find!
I also discovered the buyable ending while trying to reach a high round so that was really fun as well!

My only complaint is that after having opened every wall the map gets a bit too easy with such great wall weapons (I also got the MK2 from the box which I guess you'd want to remove if you ever update the map).
